robot_skills/src/robot_skills/picovoice.py

```diff
@@ -0,0 +1,40 @@
+import rospy
+
+from hmi.client import TimeoutException
+from hmi.common import HMIResult
+from picovoice_msgs.msg import GetIntentAction, GetIntentGoal, GetIntentResult
+from robot_skills.robot_part import RobotPart
+
+
+class PicoVoice(RobotPart):
+    def __init__(self, robot_name: str, tf_buffer):
+        super().__init__(robot_name=robot_name, tf_buffer=tf_buffer)
+        self._action_client = self.create_simple_action_client(f"/{robot_name}/get_intent", GetIntentAction)
+
+    def get_intent(self, context_url: str, require_endpoint=True, timeout: float = 10.0) -> HMIResult:
+        """
+        get_intent
+
+        :param context_url:
+        :param require_endpoint:
+        :param timeout:
```

Comment:
Please add a bit of context what these do. Not all users will be familiar with picovoice workings.